src/agentscope/models/openai_model.py

```diff
@@ -55,14 +56,23 @@ def __init__(
                 The total budget using this model. Set to `None` means no
                 limit.
         """
-        super().__init__(name)
+        if model is None:
+            model = model_id
+        super().__init__(
```

Comment:
Because the initialization method of the base class will automatically print all input parameters, passing in these parameters can facilitate debugging.